framework/helpers/BaseHtml.php

```diff
@@ -1314,6 +1314,8 @@ public static function activeInput($type, $model, $attribute, $options = [])
             $options['id'] = static::getInputId($model, $attribute);
         }
 
+        self::activePlaceholder($model, $attribute, $options);
```

Comment:
I'll close my PR, let's finish this one. I don't like the idea of hidden changes of options.

Из этой строчки не видно, что метод поменяет опцию пока не посмотришь на метод и не поймешь что параметр передан по ссылке. Это не читабельно и неявные изменения имхо это плохо. В целом не вижу смысла создавать отдельный метод для этой проверки.

Reply:
In this case, you have to copy the code for textarea.

В таком случаем придется копировать код для textarea.

Reply:
At least you can rename `activePlaceholder` to `setActivePlaceholder` to give a sense that method will change something. But I still prefer to have a couple of similar lines in both methods rather than not obvious changes of parameters. Let's wait for the third opinion

Reply:
Rename method activePlaceholder to setActivePlaceholder